v3/src/data-interactive/handlers/interactive-frame-handler.ts

```diff
@@ -20,6 +20,7 @@ export const diInteractiveFrameHandler: DIHandler = {
     } = webViewContent ?? {}
     const values: DIInteractiveFrame = {
       allowEmptyAttributeDeletion,
+      codapVersion: "v3", // TODO Correct version
```

Comment:
See `handle-cfm-event.ts` for a means of retrieving the version number. Seems like there should be a means of retrieving it once and caching it.